src/serac/physics/contact/contact_data.hpp

```diff
@@ -156,13 +156,22 @@ class ContactData {
    */
   void setDisplacements(const mfem::Vector& u);
 
+#ifdef SERAC_USE_TRIBOL
   /**
    * @brief Have there been contact interactions added?
    *
    * @return true if contact interactions have been added
    * @return false if there are no contact interactions
    */
   bool haveContactInteractions() const { return !interactions_.empty(); }
+#else
```

Comment:
would it be better to put the ifdef inside of the function?

Reply:
Yeah, I was doing this quickly. I'll update that